regolith/helpers/l_generalhelper.py

```diff
@@ -7,10 +7,74 @@
     all_docs_from_collection,
     get_pi_id,
     search_collection,
-    collection_str
+    collection_str,
 )
 HELPER_TARGET = "lister"
+KEYS = {"abstracts":{"firstname": "first name of the author.",
```

Comment:
no, this is the wrong approach.  

more like
```
if rc.print_keys
    print(*[f"{key}\n" for key in rc.coll.keys()])
    return
```

Reply:
hi @sbillinge!
Does this looks better?